### ISSUE TITLE
Log flooding in airplane mode

### ISSUE THREAD

**Issue by pawi-cmrg** [getsentry/sentry-unity#576]
In a new, empty project, after adding Sentry (0.10.1), sentry will flood the logs with error messages if the device is in airplane mode (or not connected to wifi). See attached logs

We have noticed some app freezes of our app, and it's unclear if this is related. 
[sentry-flooding.log](https://github.com/getsentry/sentry-unity/files/8108541/sentry-flooding.log)

**Comment by bitsandfoxes** [comment]
Hey @pawi-cmrg, thanks for raising this. Is this some new behaviour that you did not experience in earlier versions?

**Comment by pawi-cmrg** [comment]
We first observed this in 0.9.4. Not sure if it was there before.

**Comment by bruno-garcia** [comment]
Seems to be the internal diagnostic logger of the SDK. We don't support having that turn on in production. That's just to debug the SDK internals. So if you're trying to figure out why the SDK isn't sending events to Sentry, you turn Debug mode on and see why.

**Comment by pawi-cmrg** [comment]
Ok, thanks for the clarification. I thought I saw this on production too. We did not knowingly enable it, which would be the flag to enable/disable it?

In that context, it looks like Sentry is trying to upload quite aggressively (resend after fail, leading to several attempts per second), even in airplane mode. Is there a way to add some backoff or delay to it

https://user-images.githubusercontent.com/67897834/155276070-6f7889b8-c403-4a87-9518-0775dbf30355.mov

?
